collectors/python.d.plugin/mysql/README.md

```diff
@@ -331,7 +331,7 @@ This module will produce following charts (if data is available):
 2.  **Commands** in commands/s
 
     -   select
-    -   update
+    -   updater
```

Comment:
It is not `updater`, it is `update`

https://github.com/netdata/go.d.plugin/blob/359f7334899f691a26701d57b0075471b8b98a80/modules/mysql/testdata/MariaDBv10.5.4-galera-%5Buser_statistics%5D.txt#L16-L18

Reply:
nice thanks!